README.md

```diff
@@ -119,49 +119,111 @@ bootstrap VM and master nodes using the Terraform providers for libvirt
-setup on local machine:
+Accessing the web Console running on virtualized cluster from local web browser requires additional setup on local machine and the virt host to enable forwarding to cluster's VMs.
 
-Add entry to ```/etc/hosts```:
-```
-# /etc/hosts
-192.168.111.4   console-openshift-console.apps.ostest.test.metalkube.org console openshift-authentication-openshift-authentication.apps.ostest.test.metalkube.org api.ostest.test.metalkube.org prometheus-k8s-openshift-monitoring.apps.ostest.test.metalkube.org alertmanager-main-openshift-monitoring.apps.ostest.test.metalkube.org kubevirt-web-ui.apps.ostest.test.metalkube.org oauth-openshift.apps.ostest.test.metalkube.org grafana-openshift-monitoring.apps.ostest.test.metalkube.org
-```
-Run sshuttle:
+There are two ways to achieve this, by using `sshutle` or `xinetd`.
 
-```
-sshuttle -r <user>@<virthost> 192.168.111.0/24
-```
+### sshutle (works only with IPv4)
+
+1. On your local machine install `sshutle`
+   
+2. Add entry to `/etc/hosts`
+    
+    ```
+    192.168.111.4 console-openshift-console.apps.ostest.test.metalkube.org console openshift-authentication-openshift-authentication.apps.ostest.test.metalkube.org api.ostest.test.metalkube.org prometheus-k8s-openshift-monitoring.apps.ostest.test.metalkube.org alertmanager-main-openshift-monitoring.apps.ostest.test.metalkube.org kubevirt-web-ui.apps.ostest.test.metalkube.org oauth-openshift.apps.ostest.test.metalkube.org grafana-openshift-monitoring.apps.ostest.test.metalkube.org
+    ```
+
+3. Run sshuttle on the local machine
+
+    ```
+    sshuttle -r <user>@<virthost> 192.168.111.0/24
+    ```
+
+### xinetd
+
+This approach uses xinetd instead of iptables to allow IPv4 to IPv6 forwarding.
+
+1. Install xinetd
+
+    ```
+    sudo yum install xinetd -y
+    ```
+
+2. Create the xinetd config file
+
+    ```
+    mkdir -p /etc/xinetd.d/
+    ```
+
+3. Copy the example config file
+
+    ```
+    cp dev-scripts/openshift_xinetd_example.conf /etc/xinetd.d/openshift
+    ```
+
+4. Edit the config file
+
+    ##### Set `bind` to the public address of the lab machine
+
+    ##### Set the `redirect` IPs to dev-scripts values for apiVIP and ingressVIP, which can be found at `dev-scripts/ocp/.openshift_install_state.json`.
+
+    ```
+    vim /etc/xinetd.d/openshift
+    ```
+
+5. Restart xinetd
+   
+    ```
+    sudo systemctl restart xinetd
+    ```
+
+6. Populate your local machine's `/etc/hosts/` with addresses as described in the playbook output
+
+    ```
+    <hostIP> console-openshift-console.apps.<CLUSTER_NAME>.test.metalkube.org openshift-authentication-openshift-authentication.apps.<CLUSTER_NAME>.test.metalkube.org grafana-openshift-monitoring.apps.<CLUSTER_NAME>.test.metalkube.org prometheus-k8s-openshift-monitoring.apps.<CLUSTER_NAME>.test.metalkube.org api.<CLUSTER_NAME>.test.metalkube.org oauth-openshift.apps.<CLUSTER_NAME>.test.metalkube.org
```

Comment:
Why `<CLUSTER_NAME>` here? I'd use the same content of the previous section

Reply:
I would  hilight a little bit more the fact that `<hostIP>` must be replaced

Reply:
> Why `<CLUSTER_NAME>` here? I'd use the same content of the previous section

What content exactly? IP addresses?

Reply:
There's a similar content but `ostest` is used